docs/introduction/Cross-chain-transfer-protocol.rst

```diff
@@ -41,6 +61,44 @@ A backward transfer moves coins back from a sidechain to the mainchain destinati
 A Backward Transfer is initiated by a **Withdrawal Request** which is a sidechain transaction issued by the coin's owner. The request specifies the mainchain destination address and the amount. More precisely, the withdrawal request owner will create a WithdrawalRequestBox that destroys the specified amount of coins in the sidechain. This is not enough to move those coins back to the mainchain though: we need to wait until the end of the withdrawal epoch, when all the coins specified in that epoch’s Withdrawal Requests are listed in a single certificate, that is then propagated to the mainchain.
 The certificate includes a succinct cryptographic proof that the rules associated with the declared verifying key have been respected. Certificates are processed by the mainchain consensus, which recreates the coins as specified by the certificate, only checking that the proof verifies, and that the coins received by a sidechain match the amount that was sent to it.
 
+Can be performed by following the RPC command on the Mainchain side:
+
+.. code:: Bash
+
+    sc_request_transfer(outputs, params)
+
+Where outputs are an array of JSON objects representing the amounts to send. Each array element must contain the following elements
+   scid           - sidechain ID in uint256 format
+   vScRequestData - sidechain UTXO ID for the backward transfer that is being requested
+   pubkeyhash     - mainchain address that will receive the backward transferred amount
+   scFee          - value spent by the sender that will be gained by an SC forger
+
+Params is a JSON object with the following command parameters
+   fromaddress   - The address to send the funds from. If omitted funds are taken from all available UTXO.
+   changeaddress - The address to send the change to, if any. If not set, fromaddress is used. If the latter is not set, a newly generated address will be used.
+   minconf       - Minimum confirmations the funds should have.
+   fee           - The fee amount to attach to this transaction.
+
+Ceased Sidechain Withdrawal
+===========================
+
+The funds of the ceased sidechain can be retained to the mainchain with Ceased Sidechain Withdrawal request. This request can be performed right after the sidechain ceasing.
+To perform this request on the sidechain side should be generated nullifier and Ceased Sidechain Withdrawal proof. Nullifier can be generated by api command nullifier(csw api group). Proof generation can be done with generateCswProof command.
```

Comment:
To perform this request on the sidechain side should be generated nullifier and Ceased Sidechain Withdrawal proof. Nullifier can be generated by API command nullifier(CSW API group). Proof generation can be done with generateCswProof command.

I changed the capitalization of API inside and outside of the parentheses. You may need to change it back if that is a specific command.